### PR TITLE
feat: wip tooltip

### PR DESCRIPTION
**Pull request by goetzrobin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our
      guidelines: https://github.com/goetzrobin/spartan/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Which package are you modifying?

- [ ] accordion
- [ ] alert
- [ ] alert-dialog
- [ ] aspect-ratio
- [ ] avatar
- [ ] badge
- [ ] button
- [ ] calendar
- [ ] card
- [ ] checkbox
- [ ] collapsible
- [ ] combobox
- [ ] command
- [ ] context-menu
- [ ] data-table
- [ ] date-picker
- [ ] dialog
- [ ] dropdown-menu
- [ ] hover-card
- [ ] input
- [ ] label
- [ ] menubar
- [ ] navigation-menu
- [ ] popover
- [ ] progress
- [ ] radio-group
- [ ] scroll-area
- [ ] select
- [ ] separator
- [ ] sheet
- [ ] skeleton
- [ ] slider
- [ ] switch
- [ ] table
- [ ] tabs
- [ ] textarea
- [ ] toast
- [ ] toggle
- [x] tooltip
- [ ] typography

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: libs/ui/tooltip/helm/src/lib/hlm-tooltip.directive.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+			this._brnTooltip.exitAnimationDuration = 150;
+			this._brnTooltip.hideDelay = 300;
+			this._brnTooltip.showDelay = 150;
+			this._brnTooltip.tooltipContentClasses =
+				'overflow-hidden rounded-md border border-border bg-popover px-3 py-1.5 text-sm text-popover-foreground shadow-md fade-in-0 zoom-in-95 ' +
+				'data-[state=open]:animate-in ' +
+				'data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95 ' +
+				'data-[side=below]:slide-in-from-top-2 data-[side=above]:slide-in-from-bottom-2 ' +
+				'data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 ' +
+				'data-[side=after]:slide-in-from-left-2 data-[side=before]:slide-in-from-right-2 ';
```
Comment (elite-benni) [inline libs/ui/tooltip/helm/src/lib/hlm-tooltip.directive.ts:38]: How does this behave in a right-to-left layout? 
Is this handled by tailwind or is it sliding from the "wrong" side?

File: libs/ui/tooltip/helm/src/lib/hlm-tooltip.directive.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+@Directive({
+	selector: '[hlmTooltip]',
+	standalone: true,
+	hostDirectives: [
+		{
+			directive: BrnTooltipDirective,
+			inputs: [
+				'brnTooltip: hlmTooltip',
+				'aria-describedby',
+				'disabled',
```
Comment (elite-benni) [inline libs/ui/tooltip/helm/src/lib/hlm-tooltip.directive.ts:13]: Does this mean, that we are not able to disable the tooltip without disabling the host? 
I saw that you changed the input to only `disabled `in the brn in comparison to `matTooltipDisabled`.
How will we handle this, if we do not provide a seperate input for disabling only the tooltip?

Comment (goetzrobin) [reply libs/ui/tooltip/helm/src/lib/hlm-tooltip.directive.ts:13]: good point. I'll change this in a follow up PR!

File: libs/ui/tooltip/helm/src/lib/hlm-tooltip.directive.ts (hunk 1 above)
Comment (goetzrobin) [reply libs/ui/tooltip/helm/src/lib/hlm-tooltip.directive.ts:38]: you have a good point here too. I'll need to address this

Comment (goetzrobin) [reply libs/ui/tooltip/helm/src/lib/hlm-tooltip.directive.ts:38]: @elite-benni if you have time to tackle this feel free to create a PR. I am a little tied up right now and won't get to it in the next couple weeks...

Comment (elite-benni) [reply libs/ui/tooltip/helm/src/lib/hlm-tooltip.directive.ts:38]: I just added an PR, just removing the todo and `data-[side=after]` classes.
First I thought I just have to add `rtl:` selector for this.
As I tried out the solution i found out, that it is actually not a problem. 
`before` and `after` are translated to `left` and `right` with taking the rtl direction into account. 
Everything was fine. ;)

**Pull request merged**